src/Features/TreeBrowse.php

```diff
@@ -162,6 +162,7 @@ public static function getTreeCategoryList(string $itemtype, array $params): arr
 
         $cat_itemtype = static::getCategoryItemType($itemtype);
         $cat_item     = new $cat_itemtype();
+        $params['list_limit'] = 99999;
```

Comment:
IMHO, this is really a bad idea.

I did not think about it in my previous reviews, but displaying items counter on the category tree may make this feature  unusable on instances that have many items. Indeed, we already have perfomances issues on search view (see #9495), and here we will, I guess, use twice the already used memory to compute this tree.

Also, if we finally found a solution for #9495 issue, we will never be able to find a one for this tree computation, as we will always need to fetch all matching items.

So, I really think we should remove counters from this tree. We could eventually filter categories by doing a simple SQL query that fetched count of items for each category, without any additionnal condition.